contracts/superclasses/Posts.sol

```diff
@@ -130,6 +130,12 @@ contract Posts is KetlGuarded {
   }
 
   // Posts
+  function getPost(
```

Comment:
Hm, do we absolutely need this accessor? `posts` is public after all

Reply:
I think it's possible, but it doesn't pass it as a `Post` struct, but like a raw array. Using a function allows it to be returned as a `Post`

Reply:
Just checked it, for some reason `feeds.posts[x][y]` doesn't return `Post` 😅 adding `getPost` is fine then